pkg/topology/kademlia/internal/metrics/metrics.go

```diff
@@ -15,17 +15,27 @@ import (
 
 	"github.com/ethersphere/bee/pkg/shed"
 	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/libp2p/go-libp2p-core/network"
 	"github.com/syndtr/goleveldb/leveldb"
 )
 
+const ewmaSmoothing = 0.1
+
 // PeerConnectionDirection represents peer connection direction.
 type PeerConnectionDirection string
 
 const (
 	PeerConnectionDirectionInbound  PeerConnectionDirection = "inbound"
 	PeerConnectionDirectionOutbound PeerConnectionDirection = "outbound"
+)
+
+// PeerReachabilityStatus represents the peer reachability status.
+type PeerReachabilityStatus string
 
-	ewmaSmoothing = 0.1
+var (
+	PeerReachabilityStatusUnknown = PeerReachabilityStatus(network.ReachabilityUnknown.String())
```

Comment:
Not sure if we want to have dependency on `go-libp2p-core/network` in `kademlia`

It would be better to use the actual string values here and mention these are directly related to the libp2p network reachability

Reply:
Fixed.